pkg/apiclient/_.primary.swagger.json

```diff
@@ -301,6 +297,38 @@
       "items": {
         "$ref": "#/definitions/github.com.argoproj.argo_workflows.v3.pkg.apis.workflow.v1alpha1.WorkflowStep"
       }
+    },
+    "google.protobuf.Any": {
+      "type": "object",
+      "properties": {
+        "type_url": {
+          "type": "string"
+        },
+        "value": {
+          "type": "string",
+          "format": "byte"
+        }
+      }
+    },
+    "grpc.gateway.runtime.Error": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "integer"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/google.protobuf.Any"
+          }
+        },
+        "error": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
```

Comment:
overall looks good,
I'm not sure if we need to redefine these here though.
These are defined in api/openapi-spec/swagger.json

Reply:
Yes, we got the definitions not found error when opening pkg/apiclient/_.primary.swagger.json, so I copied them from the file you mentioned

Reply:
I think the _.primary.swagger.json is an auxiliary file that helps to construct api/openapi-spec/swagger.json
We should just validate api/openapi-spec/swagger.json

Reply:
Well, I only checked the swagger file for pkg/apiclient/_.primary.swagger.json before as the related issue is only to fix the error in it.
And I tried to open api/openapi-spec/swagger.json just now, there is no error.
Can you help to confirm if this PR is still needed or we can close the the issue and PR?

Reply:
@tczhao I saw this pull request is already approved, can we merge it or need some other actions from my side, kindly let me know if needed

Reply:
Hi @tczhao @alexec 
A gentle follow up with this PR, I am looking forward for your feedback 😄